### PR TITLE
Add the version files for the branch 202012

### PR DESCRIPTION
**Pull request by xumia**
#### Why I did it
Add the version file for reproducible build for 202012

#### How to verify it
make SONIC_VERSION_CONTROL_COMPONENTS=all <targets>
make freeze
Expect not version change after the version files added.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

### CODE REVIEW

File: files/build/versions/build/build-sonic-slave-stretch/versions-deb-stretch [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+libnl-genl-3-dev==3.5.0-1
+libnl-nf-3-200==3.5.0-1
+libnl-nf-3-dev==3.5.0-1
+libnl-route-3-200==3.5.0-1
+libnl-route-3-dev==3.5.0-1
+libsainps==1.5.0
+libsairedis==1.0.0
+libsaithrift-dev==0.9.4
+libswsscommon==1.0.0
+libswsscommon-dev==1.0.0
```
Comment (qiluo-msft) [inline files/build/versions/build/build-sonic-slave-stretch/versions-deb-stretch:19]: Many packages are build targets inside slave containers. By design, should we save their versions? If not, can you fix the code which generated them?

Comment (xumia) [reply files/build/versions/build/build-sonic-slave-stretch/versions-deb-stretch:19]: Hi Qi, it is not necessary to save the versions, but it is no harm, we do not need to additional actions to filter them out.

Comment (qiluo-msft) [reply files/build/versions/build/build-sonic-slave-stretch/versions-deb-stretch:19]: could you add a github issue? I know it is no harm right now.

**Review by qiluo-msft — APPROVED** [review]

**Review by daall — APPROVED** [review]
Just one question: I see that none of these files have a trailing newline - is this expected?

File: files/build/versions/build/build-sonic-slave-stretch/versions-deb-stretch (hunk 1 above)
Comment (xumia) [reply files/build/versions/build/build-sonic-slave-stretch/versions-deb-stretch:19]: I have created an issue https://github.com/Azure/sonic-buildimage/issues/6851

**Comment by xumia** [comment]
@daall , the version files are created by automation tools, it is as expected without a trailing newline. It will have no issue with or without a newline.

**Pull request merged**